README.md

```diff
@@ -200,7 +200,24 @@ Also you may rewrite some predefined environment variables defined in [Dockerfil
 
 ## ReDoc CLI
 
-[See here](https://github.com/Redocly/redoc/blob/master/cli/README.md)
+[ReDoc CLI](https://github.com/Redocly/redoc/blob/master/cli/README.md) tool is used to lint and bundle your OpenAPI definition into **zero-dependency** HTML file. It can be installed as a development dependency in your project.
```

Comment:
Historically there's "ReDoc", nowadays it's usually referred to in docs as "Redoc", and the cli as a tool/repo is prolly best linked as `redoc-cli` for consistence. But I'd like to see more input on correct branding from staff/owners, as it seems rather inconsistent across the repo and docs.

Reply:
> Historically there's "ReDoc", nowadays it's usually referred to in docs as "Redoc", and the cli as a tool/repo is prolly best linked as `redoc-cli` for consistence. But I'd like to see more input on correct branding from staff/owners, as it seems rather inconsistent across the repo and docs.

It wasn't stated cleary as to what was the right terminology, thanks for drawing my attention to that.